Added unsupported scenario

Matgard opened the pull request:
Intune customers using Apple DEP will be unable to enroll their Apple DEP devices into Intune if PTA is their primary form of authentication. Only workaround is to switch to a login method that supports legacy/basic authentication, or Apple must start supporting Modern Authentication during their DEP enrollment process.

tysonn commented:
@Rick-Anderson - This pull request will fail validation because the author value is required. I'm closing it. If the freshness date needs to be updated, pls make that change in the private repo so your changes go through validation. Thx.

Matgard committed:
Added unsupported scenario

Intune customers using Apple DEP will be unable to enroll their Apple DEP devices into Intune if PTA is their primary form of authentication. Only workaround is to switch to a login method that supports legacy/basic authentication, or Apple must start supporting Modern Authentication during their DEP enrollment process.
